superset/superset_typing.py

```diff
@@ -53,8 +53,8 @@ class AdhocMetric(TypedDict, total=False):
 
 class AdhocColumn(TypedDict, total=False):
     hasCustomLabel: Optional[bool]
-    label: Optional[str]
```

Comment:
I couldn't find a way on frontend to set columns without custom label, even if its set as blank it will send empty string.

Reply:
don't add any unrelated changes for the regression bugfix. we could open a new pr for the refactor.

Reply:
reverted